jupyter-js-widgets/src/widget_bool.js

```diff
@@ -1,42 +1,43 @@
 // Copyright (c) Jupyter Development Team.
 // Distributed under the terms of the Modified BSD License.
-"use strict";
+'use strict';
 
-var widget = require("./widget");
-var $ = require("./jquery");
-var _ = require("underscore");
+var widget = require('./widget');
+var _ = require('underscore');
 
 var BoolModel = widget.DOMWidgetModel.extend({
     defaults: _.extend({}, widget.DOMWidgetModel.prototype.defaults, {
         value: false,
-        description: "",
+        description: '',
         disabled: false,
-        _model_name: "BoolModel"
-    }),
+        _model_name: 'BoolModel'
+    })
 });
 
 var CheckboxModel = BoolModel.extend({
     defaults: _.extend({}, BoolModel.prototype.defaults, {
-        _view_name: "CheckboxView",
-        _model_name: "CheckboxModel"
-    }),
+        _view_name: 'CheckboxView',
+        _model_name: 'CheckboxModel'
+    })
 });
 
 var CheckboxView = widget.DOMWidgetView.extend({
-    render: function(){
+    render: function() {
         /**
          * Called when view is rendered.
          */
-        this.$el
-            .addClass('jupyter-widgets widget-hbox widget-checkbox');
-        this.$checkbox = $('<input />')
-            .attr('type', 'checkbox')
-            .appendTo(this.$el)
-            .click($.proxy(this.handle_click, this));
-        this.$label = $('<div />')
-            .addClass('widget-label')
-            .appendTo(this.$el)
-            .hide();
+        this.el.classList.add('jupyter-widgets');
+        this.el.classList.add('widget-hbox');
+        this.el.classList.add('widget-checkbox');
+
+        this.checkbox = document.createElement('input');
+        this.checkbox.setAttribute('type', 'checkbox');
+        this.el.appendChild(this.checkbox);
+
+        this.label = document.createElement('div');
+        this.label.classList.add('widget-label');
+        this.label.style.display = 'none';
+        this.el.appendChild(this.label);
```

Comment:
Writing HTML in JS is so awkward.  This is one of those moments I wish we were using JSX or webpack to pull in the HTML.  Maybe in a separate PR we should look into webpacking HTML...